Neos.Utility.ObjectHandling/Classes/ObjectAccess.php

```diff
@@ -376,23 +374,15 @@ public static function isPropertyGettable($object, $propertyName)
         if (!is_object($object)) {
             throw new \InvalidArgumentException('$object must be an object, ' . gettype($object) . ' given.', 1259828921);
         }
-        if ($object instanceof \ArrayAccess && isset($object[$propertyName]) === true) {
-            return true;
-        } elseif ($object instanceof \stdClass && array_search($propertyName, array_keys(get_object_vars($object))) !== false) {
+        if (($object instanceof \ArrayAccess && $object->offsetExists($propertyName)) || ($object instanceof \stdClass && array_key_exists($propertyName, get_object_vars($object)))) {
```

Comment:
Shouldn't the \stdClass case be already done in line 358 and so be superfluous here?